Restore old debug commands

ROTARTSI82 opened the pull request:
Make old debug commands work.

While messing around, I also noticed an issue: Debug keys will get overridden when loading settings created by a non-debug build.

isage commented on src/console.cpp:
```diff
@@ -101,9 +104,20 @@ static void __sound(std::vector<std::string> *args, int num)
+    "gestation", "town", "fanfale1", "balrog", "cemetary", "plant", "pulse", "fanfale2",
+    "fanfale3", "tyrant", "run", "jenka1", "labyrinth", "access", "oppression", "geothermal",
+    "theme", "oside", "heroend", "scorching", "quiet", "lastcave", "balcony", "charge",
+    "lastbattle", "credits", "zombie", "breakdow+n", "hell", "jenka2", "waterway", "seal",
```
typo

isage commented on src/console.cpp:
```diff
@@ -568,10 +582,17 @@ static void __show_textbox(std::vector<std::string> *args, int num)
+                {"player->hide",   __hide_player,    1, 1,   "Hide player <0 or 1> (default: 1)"},
+                {"player->inputs_locked", __lock_inputs, 1, 1, "Lock inputs <0 or 1> (default: 1)"},
+                {"game.frozen",    __freeze_game,    1, 1,   "Freeze game <0 or 1> (default: 1)"},
+                {"textbox.SetVisible", __show_textbox, 1, 1, "Show textbox <0 or 1> (default: 1)"},
```
can we rename those commands? `player->hide` -> `hide_player`, etc.

ROTARTSI82 commented:
I've fixed the stuff you've mentione. In 5fc5a6c, I made it so that `input_set_mappings` will never unbind mappings. I also removed unbinding the debug fly, freeze frame, and frame advance keys in release mode as I don't see a reason for doing so (they are never used anyway).
